hack/generate-deploy-scripts.sh

```diff
@@ -22,7 +22,7 @@ fi
 set -o nounset
 set -o pipefail
 
-K8S_VERSION="1.22"
+K8S_VERSION="1.26"
```

Comment:
Upgrade k8s version to support adding RollingUpdate strategy for DaemonSet in Exoscale

Reply:
Would this not have an impact on people using 1.24/1.25?

Reply:
I have the same question. Can you provide a reference for why this version needs to be upgraded?

Reply:
I was unable to generate the correct deploy files for Exoscale with 1.22 version, so I tried to upgrade to 1.26,
refer to the version here: https://github.com/kubernetes/ingress-nginx/blob/main/build/dev-env.sh#L65C1-L66C1

Just now I tried to run `hack/generate-deploy-scripts.sh` with 1.26, now it can generate the deploy yaml files correctly, weird.

Reply:
Reverted to 1.22